src/test/java/org/jabref/logic/importer/QueryConverterTest.java

```diff
@@ -0,0 +1,59 @@
+package org.jabref.logic.importer;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.jabref.logic.importer.fetcher.ComplexSearchQuery;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class QueryConverterTest {
+    QueryConverter converter = new QueryConverter();
+
+    @Test
+    public void convertAuthorField() throws Exception {
+        ComplexSearchQuery searchQuery = new QueryConverter().convertQueryStringIntoComplexQuery("author:\"Igor Steinmacher\"");
+        assertEquals(List.of("\"Igor Steinmacher\""), searchQuery.getAuthors().get());
```

Comment:
Please create a new expected complex search query, and test against this. For example, the current test also passes if `searchQuery` has a non-empty `title` because this field is not checked in this test.